testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/providers/sse/SseCORSFilterTest.java

```diff
@@ -28,14 +29,12 @@
    public static Archive<?> deploy()
    {
       WebArchive war = TestUtil.prepareArchive(SseCORSFilterTest.class.getSimpleName());
-      war.addClass(SseCORSFilterTest.class);
-      war.addAsWebInfResource("org/jboss/resteasy/test/providers/sse/filter/web.xml", "web.xml");
       war.addAsManifestResource(EmptyAsset.INSTANCE, "beans.xml");
       war.addAsManifestResource(PermissionUtil.createPermissionsXmlAsset(
               new RuntimePermission("modifyThread")
       ), "permissions.xml");
-      return TestUtil.finishContainerPrepare(war, null, SseFilterApplication.class, SseResource.class,
-            CORSFilter.class, ExecutorServletContextListener.class);
+      return TestUtil.finishContainerPrepare(war, null, Arrays.asList(SseResource.class,
+            CORSFilter.class), ExecutorServletContextListener.class);
    }
```

Comment:
Is this test still covering the scenario for https://issues.redhat.com/browse/RESTEASY-1742 ?

Reply:
This change will use the TestApplication's  instead of SseFilterApplication. It doesn't change any test behavior.